partcad-cli/src/partcad_cli/click/command.py

```
@@ -124,6 +124,7 @@ def cli(ctx, v, q, no_ansi, p, format):
         "supply",  # Actually context is needed for "quote" but for now it it is what it is
         "test",
         "update",
+        "reset"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)